source/_components/light.yeelight.markdown

```diff
@@ -62,6 +62,30 @@ devices:
           description: "Yeelight model. Possible values are `mono1`, `color1`, `color2`, `strip1`, `bslamp1`, `ceiling1`, `ceiling2`, `ceiling3`, `ceiling4`. The setting is used to enable model specific features f.e. a particular color temperature range."
           required: false
           type: string
+custom_effects:
+  description: List of custom effects to add. Check examples below
```

Comment:
Add dots at the end of each description, then it'd be consistent with other docs.